mailpoet/assets/js/src/automation/sections/templates-section.tsx

```diff
@@ -0,0 +1,33 @@
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { MailPoet } from '../../mailpoet';
+import { workflowTemplates } from '../templates/config';
+import { TemplateListItem } from '../templates/components/template-list-item';
+
+export function TemplatesSection(): JSX.Element {
+  const templates = workflowTemplates.slice(0, 3);
+
+  return (
+    <section className="mailpoet-automation-section mailpoet-section-templates">
+      <h2>{__('Start with essentials', 'mailpoet')}</h2>
+      <p>
+        {__(
+          'Choose from our list of pre-made templates and make it your own',
```

Comment:
A trailing `.` is missing.

Reply:
c4f2431fd

Reply:
After rebase: [c437320](https://github.com/mailpoet/mailpoet/pull/4461/commits/c43732067d3e0af0e1dccb831c0c226b0314d2b0)